main.go

```diff
@@ -146,6 +153,13 @@ func createScc(ctx context.Context, mgr manager.Manager) error {
 	return err
 }
 
+func deleteScc(ctx context.Context, mgr manager.Manager) error {
+
```

Comment:
I think we should check for the presence of existing KataConfig and proceed with delete when no KataConfig exist.